digits/utils/errors.py

```diff
@@ -17,3 +17,9 @@ class LoadImageError(DigitsError):
     Errors that occur while loading an image
     """
     pass
+
+class TestError(DigitsError):
+    """
+    Errors that occur while loading an image
+    """
+    pass
```

Comment:
I'd actually like to remove this file if possible. Is there a more logical place to put this class?

Also, it looks like you copied the docstring without editing it.